core/types/block.libevm.go

```diff
@@ -160,7 +148,7 @@ type NOOPBodyHooks struct{}
 // fields and their order, which we lock in here as a change detector. If this
 // breaks then it MUST be updated and the RLP methods reviewed + new
 // backwards-compatibility tests added.
-var _ = &Body{[]*Transaction{}, []*Header{}, []*Withdrawal{}}
+var _ = &Body{[]*Transaction{}, []*Header{}, []*Withdrawal{}, nil /* extra unexported type */}
```

Comment:
I don't think adding this `nil` unexported extras field here changes anything

Reply:
Agreed and it's actually necessary